opensquirrel/circuit.py

```diff
@@ -11,6 +11,7 @@
     from opensquirrel.passes.mapper import Mapper
     from opensquirrel.passes.merger.general_merger import Merger
     from opensquirrel.passes.router.general_router import Router
+    from opensquirrel.passes.validator.general_validator import Validator
```

Comment:
With the `__init__.py` files in place, the following should be possible:
```suggestion
    from opensquirrel.passes.merger import Merger
    from opensquirrel.passes.router import Router
    from opensquirrel.passes.validator import Validator
```

Reply:
Done